FirebaseSessions/Tests/TestApp/macOS/macOS.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to push in entitlements file to the repository?

Reply:
Yeah I think we need it - it's kindof an info.plist for the macOS app without having a separate plist for just macOS

Reply:
Got it. Thanks for clarifying.